Added my name

rohanmaheshwari430 opened the pull request:
Added my name to the README for the Design@Cornell subteam.

rohanmaheshwari430 committed:
Added my name

Added my name to the README for the Design@Cornell subteam.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/cornell-dti/DesignAtCornell?pullRequest=10) <br/>All committers have signed the CLA.

rohanmaheshwari430 commented:
Added my name to the README for the dev onboarding assignment!
### **Summary**
I added my name to the README in the Design@Cornell repository
### **Test Plan ** 
### **Notes**
### **Breaking Changes**
### **None**

- [ ] I have updated the documentation accordingly.

- [ ] My PR adds a @ts-ignore

jessfeng commented:
Congrats on your first DTI PR! 🥳 Could you add everyone's roles and format it like [this ReadME](https://github.com/cornell-dti/campus-density-backend)? We can leave the other sections empty for now (I'll probably add things on later).
